community/fzy/patches/multiselect.patch

```diff
@@ -0,0 +1,1872 @@
+From 88159451c304ab45e0429b15a7a8a54a4cf560bd Mon Sep 17 00:00:00 2001
+From: leo-arch <leonardoabramovich2@gmail.com>
+Date: Sun, 17 Apr 2022 04:04:05 -0300
+Subject: [PATCH] Add multi-select and list padding
```

Comment:
This patch has much more than just multi select, it adds everything from the PR. I suggest switching to the forked repo instead and naming the package appropriately (not `fzy` since the patch changes a ton of upstream stuff)

Reply:
I know. @Vouivre wanted `leo-arch`'s features from his PR.
@dilyn-corner said in #1211 that we should use upstream, and add patches for what we wanted to add. untill they get merged or w/e.
hence the above result.

ok, so switch source to `leo-arch`'s repo, and rename.
and what would be appropriate naming here? `fzy-leo-arch`?

Reply:
`fzy-leo-arch` is fine

Reply:
done in c2ec9bb4.
all done?